packages/path_provider/path_provider_android/test/path_provider_android_test.dart

```diff
@@ -98,13 +115,27 @@ void main() {
       });
     } // end of for-loop
 
-    test('getDownloadsPath fails', () async {
-      try {
-        await pathProvider.getDownloadsPath();
-        fail('should throw UnsupportedError');
-      } catch (e) {
-        expect(e, isUnsupportedError);
-      }
+    test('getDownloadsPath succeeds', () async {
+      final String? path = await pathProvider.getDownloadsPath();
+      expect(path, kExternalStoragePaths);
+    });
+
+    test('getDownloadsPath fails with null', () async {
```

Comment:
I would not describe these as "fails"; they are returning empty lists as expected for those cases.